app/bundles/EmailBundle/Helper/MailHelper.php

```diff
@@ -1264,7 +1264,7 @@ public function getIdHash()
     public function setIdHash($idHash = null, $statToBeGenerated = true)
     {
         if ($idHash === null) {
-            $idHash = uniqid();
+            $idHash = str_replace('.','',uniqid('',true));
```

Comment:
Thanks! One more thing. The code style check is failing:

https://travis-ci.org/mautic/mautic/jobs/381897684#L706

It suggests to put some spaces after the commas like this:

```
$idHash = str_replace('.', '', uniqid('', true));
```
Can you please update?